waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -0,0 +1,921 @@
+;;
+;;       Copyright (c) 2018 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(ns waiter.scheduler.kubernetes-test
+  (:require [clj-time.core :as t]
+            [clojure.core.async :as async]
+            [clojure.data]
+            [clojure.data.json :as json]
+            [clojure.pprint]
+            [clojure.string :as str]
+            [clojure.test :refer :all]
+            [clojure.walk :as walk]
+            [plumbing.core :as pc]
+            [slingshot.slingshot :as ss]
+            [waiter.scheduler.kubernetes :refer :all]
+            [waiter.scheduler :as scheduler]
+            [waiter.util.date-utils :as du])
+  (:import waiter.scheduler.kubernetes.KubernetesScheduler
```

Comment:
Nitpick: Can we place this below `(waiter.scheduler...)`?

Reply:
Nitpick: Can we use `(waiter.scheduler.kubernetes KubernetesScheduler)` instead?

Reply:
Done.